<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseMenu</class>
 <widget class="QWidget" name="PauseMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resumeButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/icons/cat_cool.png</normaloff>:/icons/cat_cool.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="seMuteButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SEMute</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicMuteButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>MusicMute</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
